### PR TITLE
User/marcelo li gonzales/recruit training

### PR DESCRIPTION
**Pull request by MarceloLiGonzales**
Request to review Basic Training

### CODE REVIEW

File: Recruit-Training/Basic-Recruit-Training/pointers/main.cpp [hunk 1]
```diff
@@ -19,29 +20,35 @@ int main()
     int *p = &x;
 
     //TODO in the line below predict what what is going to be output
-    cout << "Predicted value of p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of p: 0xsomething" /* << PLACE YOUR PREDICTION HERE*/ << endl;
```
Comment (duan-le) [inline Recruit-Training/Basic-Recruit-Training/pointers/main.cpp:23]: You can remove the comment  `/* << PLACE YOUR PREDICTION HERE*/`. Do the same to all the other lines that this applies to.

**Review by duan-le — CHANGES_REQUESTED** [review]

File: Recruit-Training/Basic-Recruit-Training/person/main.cpp [hunk 2]
```diff
@@ -79,7 +85,7 @@ int main()
 // Pass a and b into mySwap
```
Comment (duan-le) [inline Recruit-Training/Basic-Recruit-Training/person/main.cpp:85]: Call the respective functions here.

**Review by duan-le — CHANGES_REQUESTED** [review]

File: Recruit-Training/Basic-Recruit-Training/person/main.cpp [hunk 3]
```diff
@@ -79,7 +85,7 @@ int main()
...
 // Pass in a to myIncrement
```
Comment (duan-le) [inline Recruit-Training/Basic-Recruit-Training/person/main.cpp:86]: Call this one too.

**Review by duan-le — CHANGES_REQUESTED** [review]

File: Recruit-Training/Basic-Recruit-Training/person/Person.h [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #pragma once
-
-#include <string>
+#include <iostream>
```
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/person/Person.h:2]: Why are you adding this? From the instructions, you are not permitted to modify anything in `Person.h`

File: Recruit-Training/Basic-Recruit-Training/pointers/main.cpp [hunk 5]
```diff
@@ -19,29 +20,35 @@ int main()
     int *p = &x;
 
     //TODO in the line below predict what what is going to be output
-    cout << "Predicted value of p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of p: 0xsomething" << endl;
```
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/pointers/main.cpp:23]: As mentioned in the instructions, please place your predicted values between `<< <<`.

File: Recruit-Training/Basic-Recruit-Training/pointers/main.cpp [hunk 6]
```diff
@@ -55,4 +62,11 @@ void var(int *a, int b)
 {
     *a = 365;
     b = 912;
+}
+
+void MySwap(int *a, int *b)
```
Comment (JessieG123) [inline Recruit-Training/Basic-Recruit-Training/pointers/main.cpp:67]: First letter should not be capitalized.

**Review by JessieG123 — CHANGES_REQUESTED** [review]

**Review by JessieG123 — CHANGES_REQUESTED** [review]
As mentioned in the instructions, you are to create a `Person.cpp` file. Please ensure you read the instructions properly.

**Review by duan-le — APPROVED** [review]
Everything looks good to me. Please **DO NOT** merge this branch into `master`. Get an approval from another reviewer and then you can close this PR.

**Review by JessieG123 — APPROVED** [review]
Looks good! Please **do not** merge instead just close the PR. If you need help please let us know.

**Pull request closed without merging**